.github/workflows/alchemist.yml

```diff
@@ -55,7 +55,7 @@ jobs:
         env:
           QUAY_USERNAME: ${{ secrets.QUAY_USERNAME }}
           QUAY_PASSWORD: ${{ secrets.QUAY_PASSWORD }}
-      - if: always()
+      - if: always() && github.ref == 'refs/heads/master'
```

Comment:
So does this not need the always part anymore? Why does an always exist anyway for conditionals like this?

Reply:
the always forces it to always run, even if failure, so it does need to be there

Reply:
Oh that's interesting